src/Esfa.Vacancy.Api.Types/ApprenticeshipVacancy.cs

```diff
@@ -190,7 +190,7 @@ public class ApprenticeshipVacancy
         /// <summary>
         /// The training provider's trading name
         /// </summary>
-        public string TrainingProvider { get; set; }
+        public string TrainingProviderName { get; set; }
```

Comment:
I like this but just mention it to @mgwilliam because he was cautious about changing the names of the fields that are returned. He was in favor of keeping them the same as the SOAP API

Reply:
i'm fine with this - it's very clear what the 3 provider fields refer to